mux.go

```diff
@@ -39,6 +43,10 @@ func NewRouter() *Router {
 type Router struct {
 	// Configurable Handler to be used when no route matches.
 	NotFoundHandler http.Handler
+
+	// Configurable Handler to be used request has method mismatch with route
```

Comment:
Change to "Configurable Handler to be used when the request method does not match the route."